components/shapediver/ui/DrawingOptionsComponent.tsx

```diff
@@ -0,0 +1,276 @@
+			planeRestrictionApis.forEach(r => {
+				(r as PlaneRestrictionApi).angularRestrictionApi.angleStep = Math.PI / angleStep;
+			});
+		}
+	}, [angleStep, drawingToolsApi]);
+
+	useEffect(() => {
+		if (drawingToolsApi) {
+			const geometryRestrictionApis = Object.values(drawingToolsApi.restrictions).filter(r => r instanceof GeometryRestrictionApi);
+			geometryRestrictionApis.forEach(r => {
+				(r as GeometryRestrictionApi).snapToVertices = snapToVertices;
+				(r as GeometryRestrictionApi).snapToEdges = snapToEdges;
+				(r as GeometryRestrictionApi).snapToFaces = snapToFaces;
+			});
+		}
+	}, [snapToVertices, snapToEdges, snapToFaces, drawingToolsApi]);
+
+	useEffect(() => {
+		if (drawingToolsApi) {
+			const planeRestrictionApis = Object.values(drawingToolsApi.restrictions).filter(r => r instanceof PlaneRestrictionApi);
+			setHasPlaneRestriction(planeRestrictionApis.length > 0);
+			const geometryRestrictionApis = Object.values(drawingToolsApi.restrictions).filter(r => r instanceof GeometryRestrictionApi);
+			setHasGeometryRestriction(geometryRestrictionApis.length > 0);
+		}
+	}, [drawingToolsApi]);
+
+
+
+	/**
+	 * The description of the drawing tools.
+	 * This description is shown when hovering over the info button.
+	 */
+	const description =
+		<List listStyleType="none" fw={400} size="xs">
+			<List.Item>Adding Points:
+				<List listStyleType="disc" fw={400} size="xs" fs="italic">
+					<List.Item>Starts automatically if no points exist</List.Item>
+					<List.Item>Press Insert to add a point at cursor position</List.Item>
+					<List.Item>Hover over a line segment to add a new point at the center</List.Item>
+				</List>
+			</List.Item>
+			<List.Item>Removing Points:
+				<List listStyleType="disc" fw={400} size="xs" fs="italic">
+					<List.Item>Select points and press Delete</List.Item>
+				</List>
+			</List.Item>
+			<List.Item>Moving Points:
+				<List listStyleType="disc" fw={400} size="xs" fs="italic">
+					<List.Item>Drag individual points or select multiple points to move together</List.Item>
+					<List.Item>Movement restrictions:
+						<List listStyleType="disc" fw={400} size="xs" fs="italic">
+							<List.Item>Press g for grid</List.Item>
+							<List.Item>Press a for angles</List.Item>
+							<List.Item>Press x/y/z for axes</List.Item>
+						</List>
+					</List.Item>
+				</List>
+			</List.Item>
+			<List.Item>History of Operations:
+				<List listStyleType="disc" fw={400} size="xs" fs="italic">
+					<List.Item>Press Ctrl+z to undo</List.Item>
+					<List.Item>Press Ctrl+y to redo</List.Item>
+				</List>
+			</List.Item>
+			<List.Item>Update/Cancel:
+				<List listStyleType="disc" fw={400} size="xs" fs="italic">
+					<List.Item>Press Confirm or Enter to send changes to the server</List.Item>
+					<List.Item>Press Cancel or Escape to discard changes</List.Item>
+				</List>
+			</List.Item>
+		</List>;
```

Comment:
@MajorMeerkatThe3rd it might be easier to use markdown here, and use our Markdown component. If so, the styling according to the theme will get applied automatically. There are some examples of this in the `AppBuilderPage` component.